src/classes/TssSession.class.ps1

```diff
@@ -19,15 +19,18 @@
     }
 
     [boolean]IsValidToken() {
-        if ([string]::IsNullOrEmpty($this.AccessToken)) {
+        if ($this.TokenType -like 'DefaultCredentials') {
+            Write-Host 'Token is using currently logged on user credentials'
```

Comment:
A token is not actually used with winauth. We can just have this state something like:

```
Write-Host 'Windows Integrated Auth in use
```

Reply:
in 7614719